### PR TITLE
Ensure that only group members are able to announce to a group

### PR DESCRIPTION
**Pull request by nathanbegbie**
milton-proofing the announce system

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@smn small review

**Comment by nathanbegbie** [comment]
@smn small review

**Comment by nathanbegbie** [comment]
@smn small review pls

File: announce/tests/test_bellman.py [hunk 1]
```diff
@@ -241,15 +249,15 @@ def setUp(self):
...
         group_name, space, text = bm.text.partition(' ')
 
         with patch.object(bm, 'send_announcement') as mock:
             bm.execute()
-            mock.assert_called()
+            mock.assert_called_with()
             self.assertEqual(bm.text, '\n'.join([
-                'Message from <@test_id> :',
-                '<@1> <@2> ',
+                'Message from <@1> :',
```
Comment (smn) [inline announce/tests/test_bellman.py:260]: Please add the group that's being announced to here as well?
I can be part of multiple groups and an announcement like "meeting postponed" without group context is bound to cause trouble.

**Comment by smn** [comment]
:+1: other than the minor bit about including the group being addressed in the announced message.

File: announce/tests/test_bellman.py (hunk 1 above)
Comment (nathanbegbie) [inline announce/tests/test_bellman.py:260]: Yep, being taken care of in issue#15

**Pull request merged**